projects/rocblas/clients/common/CMakeLists.txt

```
@@ -170,6 +163,9 @@ target_compile_definitions( rocblas_clients_common PRIVATE ROCBLAS_REFERENCE_LIB
 
 # target is not linked as OBJECT library but we want any interface flags from dependencies, e.g. COMMON_LINK_LIBS may bring OpenMP
 target_link_libraries( rocblas_clients_common roc::rocblas hip::host hip::device $<IF:$<TARGET_EXISTS:GTest::gtest>,GTest::gtest,GTest::GTest> $<IF:$<TARGET_EXISTS:GTest::gtest_main>,GTest::gtest_main,GTest::Main> ${COMMON_LINK_LIBS} )
+if(BUILD_WITH_HIPBLASLT)
+    target_link_libraries( rocblas_clients_common roc::hipblaslt )
```

Is there are reason that we don't provide linkage here?

---

You mean
```suggestion
    target_link_libraries( rocblas_clients_common PRIVATE roc::hipblaslt )
```
? I think the only reason might be that it was missed in the original patch.

---

PRIVATE at least, BUILD_INTERFACE I thought was fine too, as we don't want hipblaslt in our interface.